buildout.cfg

```diff
@@ -15,9 +15,9 @@ eggs =
 defaults = ['-c', '--module', 'five.formlib']
 
 [sources]
-Zope2 = git https://github.com/zopefoundation/Zope
+Zope = git https://github.com/zopefoundation/Zope
 
 [versions]
-Zope2 =
+Zope =
 mr.developer = 1.38
-zope.formlib = 4.4
+zope.formlib = >= 4.4
```

Comment:
Maybe this should be moved to `setup.py` as install constraint.

Reply:
That was already present. I deleted the whole version section.